index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>NeoDev UI</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="index.css">
+  </head>
+
+  <body>
+    <div id="app" class="font">
+      <div class="header">
+        <div class="header_logo">
+          <a href="#"><h1>NeoDev UI</h1></a>
+        </div>
+        <div class="header_links">
+          <a class="header_link" href="#">Documentation</a>
+          <a class="header_link" href="#">Github</a>
+        </div>
+      </div>
+      <div class="main">
+        <div class="main_content">
+          <h1>NeoDev UI</h1>
+          <p>
+            NeoDev UI is a Component Library that gives you the building block
+            you need to build your own application
+          </p>
+          <button class="font">Get Started</button>
+        </div>
+        <img class="main_image" src="./images/doc.svg" alt=""/>
+      </div>
+      <div class="footer">
+        <img src="./images/github-sign.png" alt="" />
```

Comment:
You can use fontawesome images for this so you can change the images color according to your theme

Reply:
Great suggestion!